.gitignore

```diff
@@ -2,4 +2,5 @@
 .venv
 .pytest_cache/
 __pycache__
-.vscode
\ No newline at end of file
+.vscode
+.DS_Store
```

Comment:
This is not related to the current PR, it would be great to have it in its own PR so we can keep track of the changes to `.gitignore`.

Reply:
Ok, No problem.
I'll revert this change.